components/raftstore/src/store/fsm/life.rs

```diff
@@ -68,12 +69,17 @@ pub fn handle_tombstone_message_on_learner<EK: KvEngine>(
     let region_state_key = keys::region_state_key(region_id);
     let local_state: RegionLocalState = match engine.get_msg_cf(CF_RAFT, &region_state_key) {
         Ok(Some(s)) => s,
-        e => panic!(
-            "[store {}] failed to get regions state of {:?}: {:?}",
-            store_id,
-            msg.get_region_id(),
-            e
-        ),
+        e => {
+            warn!(
+                "[store {}] failed to get regions state of {:?}: {:?}",
+                store_id,
+                msg.get_region_id(),
+                e
+            );
+            // A peer may never be created if its parent peer skips split by
```

Comment:
> // A peer may never be created if its parent peer skips split by

Do you mean the peer has not applied the snapshot and thus no region_state_key.  But the peer object itself is created otherwise it won't go to this path.

Reply:
This is not the path of PeerFsmDelegate but StoreFsmDelegate.